### ISSUE TITLE
the ASF module in DBNetcpp

### ISSUE THREAD

**Issue by xiongzl** [open-mmlab/mmocr#1074]
where could i find the source code of the ASF module in DBNetcpp?   have you test the  ASF module  effect ，does It has the same effect as the paper ? what‘s more , is DBNetcpp code just add ASF module compare with dbnet，have you changed other module?

**Comment by xiongzl** [comment]
i find it
Add Adaptive Scale Fusion module in mmocr/models/textdet/necks/fpn_cat.py

**Comment by gaotongxiao** [comment]
And we've reported the performance of DBNet and DBNet++ in the [docs](https://mmocr.readthedocs.io/en/latest/textdet_models.html#dbnetpp)

**Comment by xiongzl** [comment]
![image](https://user-images.githubusercontent.com/13463574/173830747-a77b7ecf-b438-4d11-9c16-3cd42dfdd8a4.png)
![image](https://user-images.githubusercontent.com/13463574/173831028-a81063fc-3b8f-4d0f-8178-30fc7082f606.png)
when i  contrast your code with the code of [MhLiao](https://github.com/MhLiao/DB/blob/master/decoders/feature_attention.py), i have some doubts? 

1. your code module is match  as i show in the first picture?
2. as the paper mentioned in ASF module. Firstly, we concatenate the scaled input features X and then a 3 × 3 convolutional layer is followed to  obtain an intermediate feature [C,H,W] .  
    the 3*3 convolutional in your code  is 
            self.asf_conv = ConvModule(
                out_channels * self.num_outs,
                out_channels * self.num_outs,
                3,
                padding=1,
                conv_cfg=None,
                norm_cfg=None,
                act_cfg=None,
                inplace=False)
but MhLiao'code  is 
   self.conv = nn.Conv2d(in_channels, inter_channels, 3, padding=1)

**Comment by xiongzl** [comment]
![image](https://user-images.githubusercontent.com/13463574/173837289-fe84e8df-1b10-479c-bc6d-43482555f107.png)

**Comment by gaotongxiao** [comment]
1. Yes.
2. They are essentially the same. [`mmcv.cnn.ConvModule`](https://mmcv.readthedocs.io/en/latest/api.html#mmcv.cnn.ConvModule) is just a convenient bundle of convolutional of conv/norm/activation layers.

**Comment by xiongzl** [comment]
thanks ！ I have viewed the  source code of  ConvModule， which bundles conv/norm/activation layers.
